README.md

```diff
@@ -229,6 +229,13 @@ A table of errors thrown by the parser:
 |`err.code`|`err.args`|Description|
 |----------|----------|-----------|
 |`invalidCheckboxDefault`||Expect 0 or 1.|
+|`invalidRange`||Expect a number or an array.|
+|`invalidRangeMultipleUnits`||Two different units are defined.|
+|`invalidRangeValue`||Values in the array must be number, string, or null.|
+|`invalidRangeDefault`|Variable type|The default value of `@var range` must be a number. This error may be thrown when parsing `number` or `range` variables.|
```

Comment:
Should this mention that JSON won't parse fractions without a leading zero, e.g. `.5` throws a parsing error, while `0.5` is valid?

Reply:
Maybe we can make the JSON parser accept them?

Reply:
Yes, that's possible... we could automatically add a leading zero where necessary.